charge/client.go

```diff
@@ -188,8 +145,7 @@ func CloseDispute(id string) (*stripe.Dispute, error) {
 
 func (c Client) CloseDispute(id string) (*stripe.Dispute, error) {
 	dispute := &stripe.Dispute{}
-	err := c.B.Call("POST", stripe.FormatURLPath("/charges/%s/dispute/close", id), c.Key, nil, nil, dispute)
-
+	err := c.B.Call2("POST", stripe.FormatURLPath("/charges/%s/dispute/close", id), c.Key, nil, dispute)
```

Comment:
This is an example where we should fix `CloseDispute()` to take a real param instead of us passing a fake one internally. This change will force us to fix all of those.

Reply:
Good call. Hm, we mighttt want to consider doing that change separately though given that I think we could push this one through without it being a breaking change.